src/vl-select.js

```diff
@@ -52,6 +52,9 @@ export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelect
   }
 
   connectedCallback() {
+    if (this.dataset.required) {
+      this.setAttribute('required', 'true');
+    }
```

Comment:
Misschien beter in een _requiredChangedCallback zetten als observed attribute, zodat changes aan het attribuut doorvloeien naar het HTML attribuut required.

Reply:
fixed